BedrockServer.cpp

```diff
@@ -645,11 +709,11 @@ void BedrockServer::suppressCommandPort(bool suppress, bool manualOverride) {
     // Process accordingly
     _suppressCommandPort = suppress;
     if (suppress) {
-        // Close the command port (if open) and suppress it such that it
+        // Close the command port, and all plugin's ports.
         // won't reopen.
         SHMMM("Suppressing command port");
-        if (port)
-            closePort();
+        if (portList.size())
```

Comment:
Use `!portList.empty()` -- `.size()` will do a linear sweep over the entire list to count how many entries there are, when in fact you just want to know if it's empty.